src/families/tron/AccountBalanceSummaryFooter.js

```diff
@@ -1,118 +1,127 @@
     <ScrollView
       horizontal
       showsHorizontalScrollIndicator={false}
       style={styles.root}
     >
-      <TouchableOpacity
-        onPress={() => {
-          /** @TODO redirect to info modal */
-        }}
-        style={styles.balanceContainer}
-      >
-        <View style={styles.balanceLabelContainer}>
-          <LText style={styles.balanceLabel}>
-            <Trans i18nKey="account.availableBalance" />
-          </LText>
-          <Info size={12} color={colors.grey} />
-        </View>
-        <LText semiBold style={styles.balance}>
-          {spendableBalance}
-        </LText>
-      </TouchableOpacity>
-      <TouchableOpacity
-        onPress={() => {
-          /** @TODO redirect to info modal */
-        }}
-        style={styles.balanceContainer}
-      >
-        <View style={styles.balanceLabelContainer}>
-          <LText style={styles.balanceLabel}>
-            <Trans i18nKey="account.tronPower" />
-          </LText>
-          <Info size={12} color={colors.grey} />
-        </View>
-
-        <LText semiBold style={styles.balance}>
-          {tronPower}
-        </LText>
-      </TouchableOpacity>
-      <TouchableOpacity
-        onPress={() => {
-          /** @TODO redirect to info modal */
-        }}
-        style={styles.balanceContainer}
-      >
-        <View style={styles.balanceLabelContainer}>
-          <LText style={styles.balanceLabel}>
-            <Trans i18nKey="account.bandwidth" />
-          </LText>
-          <Info size={12} color={colors.grey} />
-        </View>
-
-        <LText semiBold style={styles.balance}>
-          {formatedBandwidth || "–"}
-        </LText>
-      </TouchableOpacity>
-      <TouchableOpacity
-        onPress={() => {
-          /** @TODO redirect to info modal */
-        }}
-        style={styles.balanceContainer}
-      >
-        <View style={styles.balanceLabelContainer}>
-          <LText style={styles.balanceLabel}>
-            <Trans i18nKey="account.energy" />
-          </LText>
-          <Info size={12} color={colors.grey} />
-        </View>
-        <LText semiBold style={styles.balance}>
-          {formatedEnergy || "–"}
-        </LText>
-      </TouchableOpacity>
+      <InfoModal
+        isOpened={!!infoName}
+        onClose={onCloseModal}
+        data={infoName ? infoCandidates[infoName] : []}
+      />
+
+      <InfoItem
+        title={t("account.availableBalance")}
+        onPress={onPressInfoCreator("available")}
+        value={spendableBalance}
+      />
+      <InfoItem
+        title={t("account.tronFrozen")}
+        onPress={onPressInfoCreator("frozen")}
+        value={tronPower}
+      />
+      <InfoItem
+        title={t("account.bandwidth")}
+        onPress={onPressInfoCreator("bandwidth")}
+        value={formattedBandwidth.toString() || "–"}
+      />
+      <InfoItem
+        title={t("account.energy")}
+        onPress={onPressInfoCreator("energy")}
+        value={formattedEnergy.toString() || "-"}
```

Comment:
I imagine this is consistent with desktop? I think this should be a formatter in live-common. who knows if we don't want to actually format it in better ways, like with commas? just the fact we have `–` kinda make me wonder maybe we want to factorize this 🤔

Reply:
we'll discuss this in other issue 👍